helm-chart/renku/values.yaml

```diff
@@ -326,67 +326,101 @@ redis:
   ### https://github.com/bitnami/charts/tree/master/bitnami/redis         ###
   ###########################################################################
 
+  # the values below were taken from the default values for the chart
+  # and adapted to our context. The following contains a mix of specific
+  # customisations plus some higher level defaults which have been
+  # retained here so someone can get some overview of what this values
+  # file can do. 
+
   fullnameOverride: renku-redis
-  redisPort: 6379
-
-  ## config for newer redis versions, here for later reference
-  # replicas:
-  #   replicaCount: 2
-  # auth:
-  #   enabled: true
-  #   sentinel: true
-  #   existingSecret: redis-secret
-  #   existingSecretPasswordKey: redis-password
-
-  usePassword: true
-  existingSecret: redis-secret
-  existingSecretPasswordKey: redis-password
-
-  master:
-    # Fixed redis bug prevents us from using this before
-    # we bump the redis chart version
-    # spreadConstraints:
-    #   - maxSkew: 1
-    #     topologyKey: node
-    #     whenUnsatisfiable: DoNotSchedule
-    #     labelSelector:
-    #       matchLabels:
-    #         app: redis
-    persistence:
-      enabled: false
+
+  # default values in this section
+  image:
+    registry: docker.io
+    repository: bitnami/redis
+    tag: 7.0.7-debian-11-r7
```

Comment:
why pin the tag here? Should it not come from the chart?

Reply:
I think we don't have complete consistency here in the chart: the specific image is configured for postgres [here](https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/renku/values.yaml#L293). For redis, I think it's sufficient that when we do a chart upgrade we should add a comment in this values file indicating what the default redis version is for that chart version. I will modify.

Reply:
I don't doubt we have inconsistencies ;)  I suppose ideally we would take what comes from the chart, otherwise it's easy to get into unexpected situations.